app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormBranch.tsx

```diff
@@ -235,6 +240,47 @@ export const FormBranch = ({
           />
         )}
       </Form.Group>
+      <Form.Group
+        className="px-3 pt-2 border-top"
+        data-testid="screenshots-edit"
+      >
+        <Form.Row className="my-3">
+          <Col>Screenshots</Col>
+          <Form.Group as={Col} className="align-top text-right">
+            <Button
+              data-testid="add-screenshot"
+              variant="outline-primary"
+              size="sm"
+              onClick={onAddScreenshot}
+            >
+              + Add screenshot
+            </Button>
+          </Form.Group>
+        </Form.Row>
+        {branch.screenshots?.map((screenshot, idx) => (
+          <FormScreenshot
+            key={idx}
+            {...{
+              fieldNamePrefix: `${fieldNamePrefix}.screenshots[${idx}]`,
+              onRemove: () => onRemoveScreenshot!(idx),
+              defaultValues: defaultValues.screenshots?.[idx] || {},
+              setSubmitErrors,
+              //@ts-ignore react-hook-form types seem broken for nested fields
+              submitErrors: (submitErrors?.screenshots?.[idx] ||
+                {}) as FormScreenshotProps["submitErrors"],
+              //@ts-ignore react-hook-form types seem broken for nested fields
+              errors: (errors?.screenshots?.[idx] ||
+                {}) as FormScreenshotProps["errors"],
+              //@ts-ignore react-hook-form types seem broken for nested fields
+              touched: (touched?.screenshots?.[idx] ||
+                {}) as FormScreenshotProps["touched"],
+              //@ts-ignore react-hook-form types seem broken for nested fields
+              reviewErrors: (reviewErrors?.screenshots?.[idx] ||
+                {}) as FormScreenshotProps["reviewErrors"],
+            }}
+          />
+        ))}
```

Comment:
Can we flow these into separate rows for each experiment instead of packing horizontally? 🙏

Reply:
Tweaked the form to go vertically, though the screenshots take up a lot more scrolling now. That was part of why I tried horizontally. But, it does more closely match the eventual appearance on the Summary page, so that's probably a better situation